secretmanager/src/main/java/secretmanager/regionalsamples/AccessRegionalSecretVersion.java

```diff
@@ -59,22 +64,24 @@ public static void accessRegionalSecretVersion(
       AccessSecretVersionResponse response = client.accessSecretVersion(secretVersionName);
 
       // Verify checksum. The used library is available in Java 9+.
-      // If using Java 8, you may use the following:
+      // For Java 8, use:
       // https://github.com/google/guava/blob/e62d6a0456420d295089a9c319b7593a3eae4a83/guava/src/com/google/common/hash/Hashing.java#L395
       byte[] data = response.getPayload().getData().toByteArray();
       Checksum checksum = new CRC32C();
       checksum.update(data, 0, data.length);
       if (response.getPayload().getDataCrc32C() != checksum.getValue()) {
         System.out.printf("Data corruption detected.");
-        return;
+        return null;
```

Comment:
Instead of return null, throw an exception here.

Reply:
Fixed